crates/contracts/build.rs

```diff
@@ -837,6 +1273,38 @@ fn main() {
                     deployment_information: Some(DeploymentInformation::BlockNumber(13191709)),
                 },
             )
+            .add_network(
```

Comment:
AFAICS we don't use this contract anymore in the code base.
Could you check if there are other contracts that can be removed from here?

Reply:
These actually get used in the `solvers` crate. 😅